prebuilt/core/booking.json

```diff
@@ -827,7 +827,49 @@
         "MODE_RAIL": {
           "$schema": "http://json-schema.org/draft-04/schema#",
           "description": "Schema for MODE_RAIL meta field",
-          "type": "object"
+          "type": "object",
+          "properties": {
+            "outward": {
```

Comment:
Is that from (departure) station?

Reply:
Hmm i should have both departure and arrival station in `outward` and `return`

Reply:
I thought so.